### PR TITLE
Add Wikiwand presence

### LINKED ISSUE #1735: Wikiwand | wikiwand.com
**Prerequisites and essential questions** 
- [X] Is it a popular site?
- [X] Is the website older than 2 months? 
- [ ] Is the site locked to a specific country/region?
- [ ] Is the site a paid service? (e.g. Netflix, Hulu)
- [ ] Does the website feature NSFW content? (e.g. porn, etc...)
- [ ] Are you a donator/patron?
- [x] Do you acknowledge that coding presences is completely voluntary and may take time for your service to be added regardless of priority?

**What's your Discord username?** Flare#9202 

**What's the name of the service?** Wikiwand 

**What should the Presence display?** Wikiwand, article and time used to view article 

**If possible, please provide a logo for the service (512x512 minimum)**  
image

### PR DESCRIPTION
**Pull request by Hans5958**
*See also: #1735, Hans5958/PreMiD-Presences-Personal#2*

## Preword

This is the presence for Wikiwand (www.wikiwand.com), a web app for viewing Wikipedia articles/pages better. I think that's it.

This PR resolves #1735.

## Notes

- Images are from the press kit, and the logo is manually upscaled by me.
- On an unrelated note, I actually have rewritten the Wikipedia presence, along with a plan to do all wikis from the Wikimedia Foundation. I'm not sure if the rewritten part is applicable, so I will need a discussion.

## Images

| Image                                                                                                                        | Link visited                                   |
| ---------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------- |
|  | https://www.wikiwand.com/                      |
|  | https://www.wikiwand.com/en/Discord_(software) |
|  | https://www.wikiwand.com/en/NASA               |
|  | https://www.wikiwand.com/news                  |

### CODE REVIEW

File: websites/W/Wikiwand/presence.ts [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    presenceData.state = "Terms of Service";
+  } else if (currentPath[0] === "privacy") {
+    presenceData.details = "Viewing a page";
+    presenceData.state = "Privacy Policy";
+  } else {
+    presenceData.details = "Reading a wiki page";
+    presenceData.state = title;
+  }
+
+  console.log(presenceData);
```
Comment (Bas950) [inline websites/W/Wikiwand/presence.ts:74]: remove console log

File: websites/W/Wikiwand/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    "name": "Hans5958",
+    "id": "279855717203050496"
+  },
+  "service": "Wikiwand",
+  "url": "www.wikiwand.com",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/8QkIyOM.png",
+  "thumbnail": "https://i.imgur.com/vLXp14J.jpg",
+  "color": "#F7F6F5",
+  "tags": ["encyclopedia", "informations", "wiki", "wikipedia"],
```
Comment (Alanexei) [inline websites/W/Wikiwand/dist/metadata.json:13]: Can you make `informations` say `information` instead. For grammatical sakes.

File: websites/W/Wikiwand/presence.ts (hunk 1 above)
Comment (Hans5958) [reply websites/W/Wikiwand/presence.ts:74]: Will do. Forgot about it.

File: websites/W/Wikiwand/dist/metadata.json (hunk 2 above)
Comment (Hans5958) [reply websites/W/Wikiwand/dist/metadata.json:13]: Will do. FYI, the tags are based on the Wikipedia presence.

**Pull request merged**